code/bit-array/src/test/java/ru/unn/agile/bitarray/test/BitArrayTest.java

```diff
@@ -0,0 +1,313 @@
+package ru.unn.agile.bitarray.test;
+
+import org.junit.Test;
+import ru.unn.agile.bitarray.model.BitArray;
+
+import static org.junit.Assert.*;
+
+public class BitArrayTest {
+
+    @Test
+    public void canInitBitArray() {
+        // arrange
+        BitArray btr = new BitArray(10);
+
+        // act
+        byte[] btrRawArray = btr.getRawArray();
+
+        // assert
+        assertEquals(btrRawArray.length, 2);
+    }
+
+    @Test(expected = NegativeArraySizeException.class)
+    public void canInitBitArrayWithNegative() {
```

Comment:
With expected throw it's cannot init, no? About naming.

Reply:
@Lenferd corrected